test/integration/admin_login_test.rb

```diff
@@ -20,7 +20,7 @@ def setup
 
   test 'an admin user can view home page' do
     visit root_path
-    assert page.has_content?('Coffee House')
+    assert page.has_content?('Cinema Coffee')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.